PiwikTracker-Swift/Campaign.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  Campaign.swift
+//  PiwikTracker
+//
+//  Created by Cornelius Horstmann on 09.10.16.
+//  Copyright © 2016 Mattias Levin. All rights reserved.
```

Comment:
I would probably remove these comments completely and just add descriptions for classes/structs when necessary.